### ISSUE TITLE
Indexing is ignoring all but one page

### ISSUE THREAD

**Issue by itamarst** [CloudCannon/pagefind#77]
Consider the following output, running `pagefind --verbose --source _site`:

```
Running Pagefind v0.8.1
Running in verbose mode
Running from: "/home/itamarst/devel/pythonspeed.com"
Source:       "_site"
Bundle Directory:  "_pagefind"

[Walking source directory]
Found 180 files matching **/*.{html}

[Parsing files]
Did not find a data-pagefind-body element on the site.
↳ Indexing all <body> elements on the site.
5 pages found without an <html> element. 
Pages without an outer <html> element will not be processed by default. 
...

[Reading languages]
Discovered 2 languages: en-us, unknown
  * en-us: 169 pages
  * unknown: 1 page

[Building search indexes]
Language en-us: 
  Indexed 1 page
  Indexed 824 words
  Indexed 0 filters

Total: 
  Indexed 1 language
  Indexed 1 page
  Indexed 824 words
  Indexed 0 filters

Finished in 0.250 seconds
```

It found 170-180 HTML pages it knows how to index, but it only indexes one page? And when I try to search it does indeed seem to only have one page indexed.

**Comment by bglw** [comment]
Hello! 👋 

Hmm, something looks to be slightly amiss here. Since Pagefind found two languages on your site (`en-us` and `unknown`) it is running multilingual mode and building an index for each language — some wires seem to have been crossed there though, as the single page `en-us` index logged below doesn't make sense.

Are you working on a public site that I could peek at? Otherwise I'll try reproduce this bug on my own.

One useful thing to test would be to run Pagefind again, but this time with the `--force-language en` flag to opt out of the multilingual behavior — curious to see what happens in this case.

**Comment by itamarst** [comment]
That fixed it! Thank you.

**Comment by bglw** [comment]
This is definitely a bug, though, so if you have a reproduction case available I'd love to see it!

**Comment by itamarst** [comment]
I don't actually have pages in multiple languages, so I'm not sure which page it's detecting as unknown...

**Comment by bglw** [comment]
Unknown just means that the page was `<html>` without a lang atttribute, instead of `<html lang="en-us">` or the like. 

These are _supposed_ to be merged into your main language, but something has been mixed up in that logic causing the unknown page to _replace_ the main language.

There's something amiss in my language map, I'll try track it down. I can see where, but not yet how 🙂

**Comment by bglw** [comment]
In an ideal world, you would have seen this log:

https://github.com/CloudCannon/pagefind/blob/76748ed333d5f4a74305c97c5aa55c90e985f753/pagefind/src/lib.rs#L160-L167

**Comment by bglw** [comment]
Tracked it down — this will be fixed in the next release ☺️

**Comment by bglw** [comment]
Hello! This is fixed in the [Pagefind v0.9.0](https://github.com/CloudCannon/pagefind/releases/tag/v0.9.0) release 🎉
